tests/p2p/bootstrap_test/test_restart_bootstrap.py

```diff
@@ -21,7 +22,9 @@ def test_reboot_bootstrap(init_session):
     sleep_time = 5
     session_id = init_session
     bootstrap_group_id = 'bootstrap_key'
-    ss_file_path = "/Users/amit/workspace/go-spacemesh/tests/k8s/bootstrap-w-conf-ss.yml"
+    # ss_file_path = "/Users/amit/workspace/go-spacemesh/tests/k8s/bootstrap-w-conf-ss.yml"
```

Comment:
remove comment

Reply:
fixed in 207fd37bd289dcc691ce4b4207a102bc0f009af7